apps/interactiveevents/templates/a4_candy_interactive_events/module_detail.html

```diff
@@ -1,18 +1,62 @@
+    <div class="container">
+        <div class="col-12">
+            <div class="alert" role="alert">
+                <i class="fas fa-info-circle" aria-hidden="true"></i>
                 {% trans 'During this event you can hand in questions with your smartphone' %}
+            </div>
+        </div>
+        <div class="row mx-md-4 my-4 justify-content-center">
+            {% if extra_fields.event_image %}
+                <div class="col-md-6 mb-4">
+                    <img src="/media/{{ extra_fields.event_image }}" />
                 </div>
-                <div class="mb-3">
-                    {{ module.description | transform_collapsibles | richtext }}
-                </div>
+            {% endif %}
+            <div class="col-md-6">
+                {% for phase in module.phases %}
+                    {% if phase == module.active_phase and extra_fields.live_stream %}
+                        <span class="label label--live">•{% trans ' Live now' %}</span>
+                        &nbsp;
+                    {% endif %}
+                    <span class="label">{% trans 'Interactive' %}</span>
+                    <h1>{{ module.name }}</h1>
+                    <div class="mb-2">
+                        {{ module.description | transform_collapsibles | richtext }}
+                    </div>
+                    {% if phase.is_over or phase == module.active_phase %}
+                        <div class="phase-info__item u-no-border">
+                            <div class="font-weight-bold mb-1">
+                                {% html_date phase.start_date 'DATETIME_FORMAT' %}
+                                &ndash; {% html_date phase.end_date 'DATETIME_FORMAT' %}
+                            </div>
+                            <div class="mb-5">
+                                {% if phase.description|length > 80 %}
+                                    <div id="collapsibleContent" class="collapse show u-no-transition">
+                                        {{ phase.description|truncatechars:80|richtext }}
+                                    </div>
+                                    <div id="collapsibleContent" class="collapse u-no-transition">
+                                        {{ phase.description|richtext }}
+                                    </div>
+                                    <button
+                                        class="btn--link collapsible    "
+                                        type="button" data-toggle="collapse"
+                                        data-target="#collapsibleContent"
+                                        aria-expanded="false"
+                                        aria-controls="collapseExample"
+                                    >
+                                        <span class="link--more">{% trans 'Read more' %}</span>
+                                        <span class="link--less">{% trans 'Read Less' %}</span>
```

Comment:
these links are both visible at the same time on firefox, not sure why given the styling, maybe incorrect nesting